src/xdocs/config_header.xml

```diff
@@ -143,8 +143,8 @@ line 5: ////////////////////////////////////////////////////////////////////
         </p>
         <source>
 &lt;module name=&quot;Header&quot;&gt;
-    &lt;property name=&quot;header&quot;
-              value=&quot;// Copyright (C) 2004 MyCompany\n// All rights reserved&quot;/&gt;
+  &lt;property name=&quot;header&quot;
+            value=&quot;// Copyright (C) 2004 MyCompany\n// All rights reserved&quot;/&gt;
```

Comment:
shouldn't this be indented +2?

Reply:
@rnveach So should it come under just the ending of property?

Reply:
`&lt;property` is indented to position 2, so `value=&quot;` should be indented to position 4.

Reply:
Done